runtime-modules/forum/src/lib.rs

```diff
@@ -282,6 +293,7 @@ pub trait Trait: system::Trait + timestamp::Trait + Sized {
     fn is_moderator(account_id: &Self::AccountId, moderator_id: &Self::ModeratorId) -> bool;
 
     fn calculate_hash(text: &[u8]) -> Self::Hash;
+    fn get_max_category_depth() -> Self::MaxCategoryDepth;
```

Comment:
This is one way to do it, but there is a native concept of parameters which is even more safe in that it statically guarantees that its a constant value, and it also works well with the runtime metadata system and Polkadotjs. Here is an example in the proposal system

https://github.com/Joystream/joystream/blob/master/runtime-modules/proposals/codex/src/lib.rs#L168